src/ui/Base/InitializationPlaceholder.ts

```diff
@@ -219,13 +235,13 @@ export class InitializationPlaceholder {
       $$(this.root).one(QueryEvents.queryError, () => reset());
       $$(this.root).one(QueryEvents.noResults, () => reset());
     }
+
+    return this;
   }
 
   private determineResultListPlaceholder(resultListElements: HTMLElement[]) {
     let currentLayout;
-    if (this.options.layout) {
-      currentLayout = this.options.layout;
-    } else if (resultListElements.length > 1) {
+    if (resultListElements.length > 1) {
       currentLayout = HashUtils.getValue('layout', HashUtils.getHash());
     } else {
       currentLayout = resultListElements[0].getAttribute('data-layout');
```

Comment:
Is it possible that `resultListElements.length === 0`?

Reply:
Currently, no, because it only gets executed if there are actually result list in the page.

I can add that check here though, does not hurt.